Makefile

```diff
@@ -1,35 +1,46 @@
+export KO_DATA_PATH=$(shell pwd)/cmd/manager/kodata
+
 .PHONY: all
 all: local-dev
 
 .PHONY: clean
 clean:
-	-kubectl delete -f deploy/
-	-kubectl delete -f deploy/crds/
-	-kubectl delete namespace tekton-pipelines
-
-.PHONY: local-dev
-local-dev: dev-setup
-	GO111MODULE=on operator-sdk run --local --watch-namespace "" --operator-flags '--zap-encoder=console'
+	-kubectl delete -f config/
```

Comment:
could be `ko` too here :stuck_out_tongue_closed_eyes:

Reply:
done